### PR TITLE
ADD Warning about characters with URL meaning

### PR DESCRIPTION
**Pull request by dmoranj**
Add a warning in the documentation about the use of characters with meaning for the URL decoding.

### CODE REVIEW

**Comment by fgalan** [comment]
LGTM. Great!

File: doc/manuals/user/forbidden_characters.md [hunk 1]
```diff
@@ -28,3 +28,18 @@ If your aplication needs to use these characteres, you should encode it
 using a scheme not including forbidden characters before sending the
 request to Orion (e.g. [URL
 encoding](http://www.degraeve.com/reference/urlencoding.php)).
+
+There is another set of characters that requires special care from the 
+user perspective. Are those in the following list:
```
Comment (kzangeli) [inline doc/manuals/user/forbidden_characters.md]: Are those in the following list => Namely, the ones in the following list

File: doc/manuals/user/forbidden_characters.md [hunk 2]
```diff
@@ -28,3 +28,18 @@ If your aplication needs to use these characteres, you should encode it
...
+
+-   #
+-   ?
+-   /
+-   %
+-   &
+
+These characters have special meaning in the URL interpretation, and, 
+considering there are convenience operations that use entity, type and
+attribute identifiers as part of the URL, its use should be avoided.
```
Comment (kzangeli) [inline doc/manuals/user/forbidden_characters.md]: their use

File: doc/manuals/user/forbidden_characters.md [hunk 3]
```diff
@@ -28,3 +28,18 @@ If your aplication needs to use these characteres, you should encode it
...
+-   #
+-   ?
+-   /
+-   %
+-   &
+
+These characters have special meaning in the URL interpretation, and, 
+considering there are convenience operations that use entity, type and
+attribute identifiers as part of the URL, its use should be avoided. 
+The use of this characters is perfectly safe when only standard operations
```
Comment (kzangeli) [inline doc/manuals/user/forbidden_characters.md]: these characters

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**